packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx

```diff
@@ -51,7 +50,7 @@ export const SiderailNavButton: React.FC<SiderailNavButtonProps> = ({
   const isWorkspaceActive = workspaceWindowState !== 'hidden' && workspaceIndex === 0;
   const formOpenInTheBackground = workspaceIndex > 0 || (workspaceIndex === 0 && workspaceWindowState === 'hidden');
 
-  if (layout === 'tablet') {
+  if (layout === 'tablet' || layout === 'phone') {
```

Comment:
The only thing I'll remark here is that the "phone" break point is really a breakpoint for "the designs don't go this small". If we need to accomodate devices with a width below 600px, we need to talk about redesigning many of the tablet layouts to make sense.

Reply:
We should indeed discuss the designs for tablets. Can we have this ticketed and let this in for now? We haven't had luck with tablets in the sites running o3 and we don't want to block them further. I hope this is OK with you @ibacher